package.json

```diff
@@ -62,6 +62,10 @@
     "typescript": "5.6.3",
     "typescript-eslint": "8.18.1"
   },
+  "peerDependencies": {
+    "@sbb-esta/lyne-elements": "2.1.0",
```

Comment:
renovate updates this, right? and are we using the same package.json in library output?